core/server/common/src/main/java/alluxio/master/journal/raft/RaftJournalSystem.java

```diff
@@ -338,6 +338,7 @@ private RaftClient createClient() {
         .build();
     return RaftClient.newBuilder()
         .setRaftGroup(mRaftGroup)
+        .setClientId(mClientId)
```

Comment:
Why this is needed?

Reply:
It's for easier tracking so requests from same node will have the same client id.